core/src/main/java/hudson/security/GlobalSecurityConfiguration.java

```diff
@@ -184,15 +185,13 @@ public Permission getRequiredPermission() {
         return Jenkins.ADMINISTER;
     }
 
-    public static Predicate<GlobalConfigurationCategory> FILTER = new Predicate<GlobalConfigurationCategory>() {
-        public boolean apply(GlobalConfigurationCategory input) {
-            return input instanceof GlobalConfigurationCategory.Security;
-        }
-    };
+    @Restricted(NoExternalUse.class)
+    @RestrictedSince("TODO")
+    public static Predicate<Descriptor> FILTER = input -> input.getCategory() instanceof GlobalConfigurationCategory.Security;
```

Comment:
Strictly speaking, this is an incompatible change. Are we OK with that?

Reply:
I can only see mentions of this class in core: https://github.com/search?q=org%3Ajenkinsci+GlobalSecurityConfiguration&type=Code

and manual testing didn't show any issues

Reply:
More of a question for @oleg-nenashev who wasn't happy with our recent unnecessary breakages.